app_dart/test/request_handlers/github/webhook_subscription_test.dart

```diff
@@ -2707,13 +2712,38 @@ void foo() {
         action: 'closed',
         number: issueNumber,
         merged: true,
+        baseSha: 'sha1', // Found in pre-populated commits in FakeGerritService.
       );
 
       expect(db.values.values.whereType<Commit>().length, 0);
       await tester.post(webhook);
       expect(db.values.values.whereType<Commit>().length, 1);
     });
 
+    test('Fail when pull request is closed and merged, but merged commit is not found on GoB', () async {
```

Comment:
Is the base case covered when the commit is found?

Reply:
Yeah, please refer to test `Acts on closed, cancels presubmit targets, add pr for postsubmit target create`.

Reply:
Thanks!